content/200-concepts/100-components/02-prisma-client/260-client-extensions/140-shared-extensions.mdx

```diff
@@ -7,47 +7,63 @@ tocDepth: 4
 
 <TopBlock>
 
-You can share your [Prisma Client extensions](/concepts/components/prisma-client/client-extensions) with other users, either as packages or as code snippets, and import extensions that other users create into your Prisma projects.
+You can share your [Prisma Client extensions](/concepts/components/prisma-client/client-extensions) with other users, either as packages or as modules, and import extensions that other users create into your project.
+
+If you would like to build a shareable extension, we also recommend using the [`prisma-client-extension-starter`](https://github.com/prisma/TODO) template.
 
 </TopBlock>
 
 ## Share Prisma Client extensions
 
-When you want to create extensions that will work for other users, and not just for your specific schema, use the `$all` prefix, such as [`$allModels`](/concepts/components/prisma-client/client-extensions/model#add-a-custom-method-to-all-models-in-your-schema). Extensions that use `$all` apply generally, not just to a named component of a specific schema.
+When you want to create extensions other users can use, and not tailored just for your schema, Prisma provides utilities to allow you to create shareable extensions.
 
-You do not need to use the `$all` prefix with the [`client`](/concepts/components/prisma-client/client-extensions/client) component, because the `client` component always applies generally.
+### Define an extension
 
-For example, a generic extension might take the following form:
+Use the `Prisma.defineExtension` method to make your extension shareable. You can use it to package the extension to either separate your extensions in multiple files or share it with other users as an npm package.
+
+The benefit of `Prisma.defineExtension` is that it provides strict type checks and auto completion when you define or use an extension.
```

Comment:
Is that an additional benefit?
Why do you now want to have that all the time, even when writing extensions for yourself?
Are those benefits just during the extension development? Or also for the users?

Reply:
I've updated the wording, let me know if it reads better.